components/brave_wallet_ui/common/async/__mocks__/bridge.ts

```diff
@@ -349,11 +349,6 @@ export class MockedWalletApiProxy {
     getNetworkForSelectedAccountOnActiveOrigin: async () => {
       return { network: this.selectedNetwork }
     },
-    isBase58EncodedSolanaPubkey: async (key) => {
-      return {
-        result: true
-      }
-    },
```

Comment:
Used by mocked version of `getIsBase58EncodedSolPubkey` in `common/slices/endpoints/address.endpoints.ts`